torch/_inductor/codecache.py

```diff
@@ -904,6 +905,7 @@ def _save_graph(
         shape_env = FxGraphCache._get_shape_env()
         assert shape_env is not None
         symints = FxGraphCache._filter_backed_symints(example_inputs)
```

Comment:
So, what's the plan for unbacked symints allocated in forwards that end up as inputs to the backwards graph?

Reply:
On https://github.com/pytorch/pytorch/pull/126251 @masnesral claims that unbacked symints cant affect guards

Reply:
Curiosity question: can runtime asserts be associated with the backward graph today? Or do they only show up in forward (if they show up in backward, do include them directly inside the backward graph as fx nodes?)